.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -26,6 +26,9 @@ A code block with the any trace messages.
 
 
 **Version Information**
+Type of Jira instance:
+- [ ] Jira Cloud (Hosted by Atlassian)
+- [ ] Jira Server (Self-hosted)
```

Comment:
Could you also add Data Center on this line:

```suggestion
- [ ] Jira Server or Data Center (Self-hosted)
```

https://confluence.atlassian.com/enterprise/jira-server-and-data-center-feature-comparison-953651628.html